vpn/src/main/java/com/duckduckgo/mobile/android/vpn/feature/settings/BadHealthMitigationSettingPlugin.kt

```diff
@@ -29,8 +32,10 @@ import javax.inject.Inject
 )
 class BadHealthMitigationSettingPlugin @Inject constructor(
     private val appTpFeatureConfig: AppTpFeatureConfig,
+    appHealthDatabase: AppHealthDatabase
 ) : AppTpSettingPlugin {
-    private val jsonAdapter = Moshi.Builder().build().adapter(JsonConfigModel::class.java)
+    private val jsonAdapter = Moshi.Builder().add(JSONObjectAdapter()).build().adapter(JsonConfigModel::class.java)
+    private val thresholdsDao = appHealthDatabase.appHealthTriggersDao()
```

Comment:
Should we be accessing directly a DAO rather than going through a repo or similar? :D